docs/software-development-kits/javascript/express.mdx

```diff
@@ -39,274 +41,470 @@ Using [yarn](https://yarnpkg.com):
 yarn add @aserto/aserto-node
 ```
 
-:::note
-`express@^4.0.0` is a peer dependency. Make sure it is installed in your project.
-:::
-
-## Usage
+> `express@^4.0.0` is a peer dependency. Make sure it is installed in your project.
 
-> Note: the `authorizerServiceUrl` option that is used throughout is no longer a URL, but the option name is retained for backward-compatibility. It is now expected to be a hostname that exposes a gRPC binding. Any "https://" prefix is stripped out of the value provided.
+## Authorizer
 
-### jwtAuthz middleware
+### Authorizer Client
+```ts
+interface Authorizer {
+  config: AuthorizerConfig,
+  channelCredentials: ChannelCredentials = credentials.createSsl()
+};
 
-`jwtAuthz` is an Express-compatible middleware that you can place in the dispatch pipeline of a route.
+type AuthorizerConfig = {
+  authorizerServiceUrl?: string;
+  tenantId?: string;
+  authorizerApiKey?: string;
+  authorizerCertFile?: string;
+};
+```
+```ts
+const authClient = new Authorizer({
+  authorizerServiceUrl: "authorizer.prod.aserto.com:8443",
+  authorizerApiKey: "my-authorizer-api-key",
+  tenantId: "my-tenant-id",
+}, credentials.createSsl());
```

Comment:
shouldn't we use getSSLCredentials() instead of requiring the user to import the gRPC credentials class?